Sources/BraintreeDropIn/Helpers/BTUIKViewUtil.m

```diff
@@ -257,19 +257,19 @@ + (UIWindowScene *)activeWindowScene API_AVAILABLE(ios(13.0)) NS_EXTENSION_UNAVA
 }
 
 + (BOOL)isOrientationLandscape NS_EXTENSION_UNAVAILABLE("Uses APIs (i.e UIApplication.sharedApplication) not available for use in App Extensions.") {
-    if (@available(iOS 13, *)) {
-        return UIInterfaceOrientationIsLandscape([self activeWindowScene].interfaceOrientation);
-    } else {
-        return UIInterfaceOrientationIsLandscape(UIApplication.sharedApplication.statusBarOrientation);
-    }
+#if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_13_0
```

Comment:
quick fix: indentation

Reply:
Good catch: https://github.com/braintree/braintree-ios-drop-in/pull/426/commits/65ad3142b960e0c08487923fefdb903820b92564